core/capabilities/launcher.go

```diff
@@ -253,24 +252,25 @@ func (w *launcher) addRemoteCapabilities(ctx context.Context, myDON registrysync
 		switch capability.CapabilityType {
 		case capabilities.CapabilityTypeTrigger:
 			newTriggerFn := func(info capabilities.CapabilityInfo) (capabilityService, error) {
-				if !strings.HasPrefix(info.ID, "streams-trigger") {
-					return nil, errors.New("not supported: trigger capability does not have id = streams-trigger")
-				}
-
-				codec := streams.NewCodec(w.lggr)
-
-				signers, err := signersFor(remoteDON, state)
-				if err != nil {
-					return nil, err
+				var aggregator remotetypes.Aggregator
+				if strings.HasPrefix(info.ID, "streams-trigger") {
+					codec := streams.NewCodec(w.lggr)
+
+					signers, err := signersFor(remoteDON, state)
+					if err != nil {
+						return nil, err
+					}
+
+					aggregator = triggers.NewMercuryRemoteAggregator(
+						codec,
+						signers,
+						int(remoteDON.F+1),
+						w.lggr,
+					)
+				} else {
+					aggregator = remote.NewDefaultModeAggregator(2*uint32(remoteDON.F) + 1)
```

Comment:
I think F+1 is enough here: if we've received F+1 identical responses from nodes then at least 1 honest node has sent us the trigger event and we should execute the workflow.

Reply:
cc @nolag Can you double-check the accuracy of the above? I think I have it right, but your eyes would be appreciated.

Reply:
Sure, I'll change 2F+1 to F+1 identical responses